index.html

```diff
@@ -0,0 +1,33 @@
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio project two</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <section class="home">
+      <div class="header">
+        <h1>Welcome</h1>
+        <a href="" class="menu"><i class="fas fa-bars"></i></a>    
+      </div>
+      <div class="headline">
+        <h1>Hey There. I’m Aster</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.</p>
+        <ul class="social">
+          <li><a href="https://github.com/aster-alemu"><i class="fa-brands fa-github"></i></a></li> 
+          <li><a href="https://linkedin.com/in/aster-alemu"><i class="fab fa-linkedin-in"></i></a></li>  
+          <li><a href=""><i class="fa-solid fa-hand-peace"></i></a></li> 
+          <li><a href="https://twitter.com/asteralemu"><i class="fab fa-twitter"></i></a></li>
+          <li><a href=""><i class="fa-solid fa-m"></i></a></li> 
+          <li><img src="image/Group.png" alt="hero image"></li>
+        </ul>
+      </div>
```

Comment:
As the project requirements specifically ask for, we need a `section` container to wrap the headline and not a simple `div`;

- [ ] Please replace the banal `div` with a tag with more semantic meaning: `section`; :+1: